### PR TITLE
Add docs popover for SSH Keys

### LINKED ISSUE #2211: Docs popover on SSH keys page
Not sure we have a good doc to link to, so this may be conditional on making such a doc.

### PR DESCRIPTION
**Pull request by charliepark**
Fixes #2211 

Screenshot 2024-08-14 at 2 29 16 PM

Am 100% open to revamping this copy.

### CODE REVIEW

**Comment by charliepark** [comment]
Slight tweak:
Screenshot 2024-08-14 at 2 36 05 PM

File: app/pages/settings/SSHKeysPage.tsx [hunk 1]
```diff
@@ -79,6 +81,12 @@ export function SSHKeysPage() {
     <>
       <PageHeader>
         <PageTitle icon={<Key24Icon />}>SSH Keys</PageTitle>
+        <DocsPopover
+          heading="SSH key management"
```
Comment (david-crespo) [inline app/pages/settings/SSHKeysPage.tsx:85]: I'd just say "SSH keys"

File: app/util/links.ts [hunk 2]
```diff
@@ -27,6 +27,9 @@ export const links = {
     'https://docs.oxide.computer/guides/architecture/service-processors#_server_sled',
   snapshotsDocs:
     'https://docs.oxide.computer/guides/managing-disks-and-snapshots#_snapshots',
+  sshKeysDocs: 'https://docs.oxide.computer/guides/user-settings#_ssh_keys',
+  sshKeyCreateDocs: 'https://docs.oxide.computer/guides/user-settings#create-ssh-key',
+  sshKeyManageDocs: 'https://docs.oxide.computer/guides/user-settings#manage-ssh-keys',
```
Comment (david-crespo) [inline app/util/links.ts:32]: I don't think we need to link to two sections of the same page

**Review by david-crespo — APPROVED** [review]
Looks good. Let's keep an eye on the docs site in case we shuffle the pages around before v10. Angela and I talked about that a bit at sync IIRC.

**Comment by charliepark** [comment]
Sounds good. Will also let her know about this PR.

**Pull request merged**